### PR TITLE
Modify serial number and fix some typos

### PR DESCRIPTION
**Pull request by yuexiao-wang**
**- What I did**
Modify serial number and fix some typos

**- How I did it**

**- How to verify it**

**- Description for the changelog**
1. Fix some typos
2. Modify serial numbers which begin from 0

Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

### CODE REVIEW

File: docs/reference/commandline/kill.md [hunk 1]
```diff
@@ -30,5 +30,5 @@ signal specified with option `--signal`.
 
 > **Note:**
 > `ENTRYPOINT` and `CMD` in the *shell* form run as a subcommand of `/bin/sh -c`,
-> which does not pass signals. This means that the executable is not the container’s PID 1
+> which does not pass signals. This means that the executable is not the container’s PID
```
Comment (AkihiroSuda) [inline docs/reference/commandline/kill.md]: PID 1 seems correct

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

File: docs/reference/commandline/kill.md (hunk 1 above)
Comment (yuexiao-wang) [inline docs/reference/commandline/kill.md]: PTAL
Updated this PR and thanks

File: experimental/plugins_graphdriver.md [hunk 2]
```diff
@@ -278,9 +278,9 @@ Responds with a list of changes. The structure of a change is:
 Where the `Path` is the filesystem path within the layered filesystem that is
 changed and `Kind` is an integer specifying the type of change that occurred:
 
-- 0 - Modified
-- 1 - Added
-- 2 - Deleted
+- 1 - Modified
+- 2 - Added
+- 3 - Deleted
```
Comment (thaJeztah) [inline experimental/plugins_graphdriver.md]: have you verified that these new values actually match the source code? This is not just a numbered list

Comment (yuexiao-wang) [inline experimental/plugins_graphdriver.md]: Sorry, I misunderstand it.
Updated this PR.

File: experimental/plugins_graphdriver.md [hunk 3]
```diff
@@ -269,7 +269,7 @@ Get a list of changes between the filesystem layers specified by the `ID` and
 }
 ```
 
-Responds with a list of changes. The structure of a change is:
+Respond with a list of changes. The structure of a change is:
```
Comment (yuexiao-wang) [inline experimental/plugins_graphdriver.md]: But how do we deal with the sentence as follows:

**Response**:
```
{
  "Size": 512366,
  "Err": ""
}
```

Respond with the size of the new layer in bytes.
Respond with a non-empty string error if an error occurred.

Should we use "responds"?

Comment (thaJeztah) [inline experimental/plugins_graphdriver.md]: No, I think this document is just not very consistent. Experimental docs usually don't go through a full docs-review, as they may still change

**Review by thaJeztah — APPROVED** [review]
Ok, let's merge this, we can make improvements if this feature is moved from "experimental" to "mainstream" at some point

LGTM

**Pull request merged**